yaml/runner.go

```diff
@@ -91,14 +96,24 @@ func RunTestCase(c *TestCase) {
 			})
 		}
 	}
+	// First process traces with normal spans
 	for _, trace := range expected.Traces {
-		if r.MatchesMatrixCondition(trace.MatrixCondition, trace.TraceQL) {
+		if !trace.HasExpectAbsentSpans() && r.MatchesMatrixCondition(trace.MatrixCondition, trace.TraceQL) {
 			slog.Info("searching tempo", "traceql", trace.TraceQL)
 			r.eventually(func() {
 				AssertTempo(r, trace)
 			})
 		}
 	}
+	// Then process traces with ExpectAbsent spans (shorter timeout)
+	for _, trace := range expected.Traces {
```

Comment:
I wonder if this can be implemented solely by calling the same "asset tempo" as above - and just expecting a consistent failure in every run